clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -61,6 +61,9 @@ static constexpr llvm::StringLiteral InitESIMDMethodName = "__init_esimd";
 static constexpr llvm::StringLiteral InitSpecConstantsBuffer =
     "__init_specialization_constants_buffer";
 static constexpr llvm::StringLiteral FinalizeMethodName = "__finalize";
+static constexpr llvm::StringLiteral GlibcxxFailedAssertion =
+    "__failed_assertion";
+static constexpr llvm::StringLiteral GlibcxxConfigFile = "bits/c++config.h";
```

Comment:
1) What is glibcxx -- should these be talking about Libstdcxx instead?
2) Why is libstdc++ special -- should we not also support libc++, MSVC STL, EASTL, etc?

Reply:
Hi, @AaronBallman 
Yes, it is libstdc++. I will write an email to you to explain the background and reason why we need to work on this patch.
Thanks very much for the review.

Reply:
Thank you for the off-list discussion! I think glibcxx should be renamed to libstdcxx to make it more clear, but the other part is clear to me now.